eng/pipelines/jobs/pack-sign-publish.yml

```diff
@@ -42,3 +42,15 @@ jobs:
         /p:DotNetPublishUsingPipelines=true
         /p:ThirdPartyNoticesFilePath='$(Build.SourcesDirectory)/$(_TPNFile)'
       displayName: Pack, Sign, and Publish
+    - task: powershell@2
+      displayName: Generate Build Info
+      inputs:
+        targetType: inline
+        script: |
+            New-Item "$(Build.ArtifactStagingDirectory)/buildInfo" -Type Directory
+            Copy-Item "$(Build.SourcesDirectory)/artifacts/packages/Release/Shipping/*.buildversion" "$(Build.ArtifactStagingDirectory)/buildInfo/buildversion"
```

Comment:
We may want to copy the non-shipping build version information too (there shouldn't be any collisions), but that's not necessary to do right now.